CHANGELOG/v1.7.4.md

```diff
@@ -0,0 +1,52 @@
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.26.x -> v1.30.x
+- Workload Cluster: v1.24.x -> v1.30.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Changes since v1.7.3
+## :chart_with_upwards_trend: Overview
+- 23 new commits merged
+- 6 feature additions ✨
+- 5 bugs fixed 🐛
+
+## :sparkles: New Features
+- Bootstrap: Allow CAPBK to generate JoinConfiguration discovery kubeconfig (#10842)
```

Comment:
@vishalanarase I think you missed the second one I commented: https://github.com/kubernetes-sigs/cluster-api/pull/10845#discussion_r1669701391

Reply:
Added

Reply:
Thank you!